### PR TITLE
Feature/replaced schemes with globals

### PR DESCRIPTION
**Pull request by kobizz**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: includes/widgets/accordion.php [hunk 1]
```diff
@@ -290,10 +290,7 @@ protected function _register_controls() {
 				'selectors' => [
 					'{{WRAPPER}} .elementor-accordion-icon, {{WRAPPER}} .elementor-accordion-title' => 'color: {{VALUE}};',
 				],
-				'scheme' => [
-					'type' => Schemes\Color::get_type(),
-					'value' => Schemes\Color::COLOR_1,
-				],
+				'global' => Global_Style::COLOR_PRIMARY,
```
Comment (matipojo) [inline includes/widgets/accordion.php:293]: Please change the format to an array, here and all files.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/widgets/accordion.php (hunk 1 above)
Comment (kobizz) [reply includes/widgets/accordion.php]: Done

File: assets/dev/js/editor/components/settings/base/manager.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ module.exports = elementorModules.ViewModule.extend( {
 				controlsCSS = new ControlsCSSParser( {
 					id: this.getStyleId(),
 					settingsModel: this.model,
-					context: this.getEditedView(),
+					container: this.getEditedView().getContainer(),
```
Comment (matipojo) [inline assets/dev/js/editor/components/settings/base/manager.js:109]: @kobizz Please update from `feature/global-style`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/components/settings/base/manager.js (hunk 2 above)
Comment (matipojo) [reply assets/dev/js/editor/components/settings/base/manager.js:109]: @kobizz ?

**Review by rodolphebertozzo — DISMISSED** [review]

File: includes/widgets/image-gallery.php [hunk 3]
```diff
@@ -326,7 +326,9 @@ protected function _register_controls() {
 			Group_Control_Typography::get_type(),
 			[
 				'name' => 'typography',
-				'scheme' => Schemes\Typography::TYPOGRAPHY_4,
+				'global' => [
+					'default' => Colors_And_Typography::COLOR_ACCENT,
```
Comment (matipojo) [inline includes/widgets/image-gallery.php:330]:
```suggestion
					'default' => Colors_And_Typography::TYPOGRAPHY_ACCENT,
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by rodolphebertozzo — APPROVED** [review]

**Review by rodolphebertozzo — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**